### ISSUE TITLE
Use links for references to previous/future chapters

### ISSUE THREAD

**Issue by fm2003** [sttp-book/sttp-book#58]
Instead of writing "in a future chapter, we will discuss ...", create a link to the actual chapter this refers to. This also applies to:

* "in future chapters"
* "in [the] previous chapters"
* "in a previous chapter"
* "in the previous chapter"

**Comment by mauricioaniche** [comment]
For now, it's like this because links between chapters is not easy in Gitbook. I have changed the order of chapters and structure of the book a few times already. And I suppose this can still happen while we are constantly changing the book.

If we move to Docbook, that gets better, as it supports referencing. If not, when the book is stable, we can go through all these sentences and manually refer to the right place (least preferable option).

**Comment by fm2003** [comment]
Fair enough. I am fine with postponing this. Would referencing using `directory/file.md` pose many problems, though? I did this in `contribute.md` already, for example. And it works basically the same way as links between HTML pages.

**Comment by mauricioaniche** [comment]
That would work for the website, but not for the PDF. 

For chapters that exist, the link should be stable by now. But for others not yet, e.g., I'm not happy with ethical testing in the non-functional testing section, for example.

**Comment by fm2003** [comment]
Ah yes, links to "local" Markdown files break in the PDF indeed. So it seems we'll have to turn to #56 before we can attempt to solve this issue...
